apps/minifront/src/components/header/header.tsx

```diff
@@ -3,7 +3,7 @@ import { SyncStatusSection } from './sync-status-section';
 
 export const Header = () => {
   return (
-    <header>
+    <header className='w-full bg-gradient-to-t from-transparent to-black to-40% pb-[3em]'>
```

Comment:
Not sure what this is doing... it's fading to black at the top? I can't visually see a difference.

Reply:
yes, this is for some pages (that aren't in this pr) which apply a full-page background image